[ADD] l10n_de_location_nuts addon

antespi opened the pull request:
This addon depends on oca/partner-contact/base_location_nuts, merge pending https://github.com/OCA/partner-contact/pull/107
# NUTS Regions for Germany

This module allows to import NUTS locations for Germany.
- German states (NUTS level 2) as Partner State
- German districts (NUTS level 3) as Partner Substate
- German regions (NUTS level 4) as Partner Region

After installation, you must click at import wizard to populate NUTS items
in Odoo database in:
Sales > Configuration > Address Book > Import NUTS 2013

This wizard will download from Europe RAMON service the metadata to
build NUTS in Odoo. This localization addon will inherit this wizard and
relate each German NUTS item with German state.

Only Administrator can manage NUTS list (it is not neccesary because
it is an European convention) but any registered user can read them,
in order to allow to assign them to partner object.

Commit:
[ADD] l10n_de_location_nuts

Commit:
[ADD] l10n_de_location_nuts

OSguard commented:
we need to add https://github.com/OCA/partner-contact to .travis file

antespi commented:
Please, review OCA/partner-contact#107 (and if it is correct, merge). This addon depends on base_location_nuts. That's why travis and runbot fail

antespi commented:
runbot error is because this PR OCA/partner-contact#107 is not meged yet

rafaelbn commented:
Waiting review of https://github.com/OCA/partner-contact/pull/107

rafaelbn commented:
Hi, who is the responsible of German localization?

OSguard commented:
me - i just restart runbot & travis build

rafaelbn commented:
Thanks @OSguard !

OSguard commented:
@rafaelbn please add the oca_dependencies.txt
like here: https://github.com/OCA/rma/pull/63/files to get CI green

antespi committed:
Add oca_dependencie.txt with partner-contact repo

antespi commented:
@rafaelbn @OSguard rebased and oca_dependencies.txt added
Hope this fix travis :smile:

OSguard commented:
@guewen can you help us, what do we wrong?

yajo committed:
Update for base_location_nuts 8.0.2.0.0.

Must merge https://github.com/OCA/partner-contact/pull/233 before to work.

yajo committed:
Fix some bugs found in functional testing.

Includes a workaround for https://github.com/odoo/odoo/pull/10285, hoping that
it gets merged and it can be removed for Odoo 10.

antespi commented:
This will not work properly until https://github.com/OCA/partner-contact/issues/233 is merged

antespi committed:
Hook for setting Germany country

antespi committed:
Travis fixes

antespi committed:
Spanish translation fixes

antespi committed:
Use separate and explicit fields for each NUTS level

rafaelbn commented:
Hi,

This module is ready to be merged. I have been using in production enviroment for 3 months

:+1: 

cc @pedrobaeza

pedrobaeza commented on .travis.yml:
```diff
@@ -22,6 +22,8 @@ virtualenv:
 
 install:
+  - pip install requests
+  - git clone https://github.com/OCA/partner-contact ${HOME}/partner-contact -b ${VERSION}
```
Put this in oca_dependencies.txt

pedrobaeza replied:
I see it's already put, so remove it from here.

pedrobaeza commented:
Little comment, but :+1: @OSguard, if you give your OK, I'll fix the comment when merging.

OSguard commented:
:+1: do it

antespi committed:
Remove duplicated dependency

antespi commented:
Rebase and partner-contact dependency removed from travis file

pedrobaeza commented:
We can merge this one also.
